### DISCUSSION TITLE
Is it possible to pass port forwarding via command line start?

### DISCUSSION THREAD

**Discussion by lattice0** [lima-vm/lima#1848] [Q&A]
`limactl start --help` does not show anything about port forwarding. Is there a way?

**Comment by jandubois** [comment]
You'll have to use `--set` for anything that doesn't have an explicit commandline option. It uses `yq` syntax. E.g.

```
--set '.portForwards += [{"guestPort": 8080, "hostPort": 8888}]'
```

Install `yq` and experiment from the commandline to get the syntax right before you pass it to `limactl`:

```console
$ echo '{}' | yq '.portForwards += [{"guestPort":8080, "hostPort": 8888}]'
portForwards:
  - guestPort: 8080
    hostPort: 8888
```
